libraries/tools/kotlin-stdlib-gen/src/templates/Aggregates.kt

```diff
@@ -840,6 +840,63 @@ object Aggregates : TemplateGroupBase() {
         }
     }
 
+    val f_reduceOrNull = fn("reduceOrNull(operation: (acc: T, T) -> T)") {
+        include(ArraysOfPrimitives, ArraysOfUnsigned, CharSequences)
+    } builder {
+        inline()
+        specialFor(ArraysOfUnsigned) { inlineOnly() }
```

Comment:
We can introduce this function in 1.3.70 as experimental: then we need to annotate it with `@SinceKotlin("1.3")` and `@ExperimentalStdlibApi`. Otherwise we need to set SinceKotlin to 1.4.

Reply:
Having this in the 1.3.70 would be great yeah! Added both annotation :) and pushed the changes, thanks for the review @ilya-g !

Reply:
One question, you meant both? f_reduceOrNull and f_reduceOrNullSuper right?

Reply:
Yes, both of them